src/cart/models.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth import get_user_model
+from django.core.validators import MinValueValidator
+from django.db import models
+
+
+class Cart(models.Model):
+    user = models.ForeignKey(to=get_user_model(), on_delete=models.CASCADE)
+    creation_date = models.DateTimeField(auto_now_add=True)
```

Comment:
if its type, better to use chooice

Reply:
I'm not clearly understand about choice field, do you mean some exact field to use the choice?
I'm also not sure of how to implement the customer's shop basket in database level.